api/Navigator.json

```diff
@@ -1306,7 +1306,7 @@
               "version_added": "29"
             },
             "firefox_android": {
-              "version_added": false
+              "version_added": "32"
```

Comment:
Is this based on testing Firefox for Android 29-32? Or why not 29 here?

Reply:
Support for the Gamepad API was added in Firefox Android 32 per our data, so this version number is offset to match that of the Gamepad interface.